doc-surrealdb_versioned_docs/version-1.x/security/authentication.mdx

```diff
@@ -240,4 +240,18 @@ curl -X POST \
 	-H "Accept: application/json" \
 	-d '{"NS":"test", "DB":"test", "SC":"user", "email":"john@doe.org", "password":"VerySecurePassword!"}' \
 	http://localhost:8000/signin
-```
\ No newline at end of file
+```
+
+## Sessions
+
+Whenever authentication is performed with any kind of user against SurrealDB, a session is established between the client and the SurrealDB server with which the connection was established. These sessions persist on the server for the duration of the connection, whether it is a single request through the [HTTP REST API](/docs/surrealdb/integration/http) or through multiple requests in the same connection using the [WebSocket API](/docs/surrealdb/integration/rpc) and any of the [SDK](/docs/surrealdb/integration/sdks/) that leverage it.
```

Comment:
1) Recommend pluralizing SDK to SDKs
2) So through the HTTP REST API the sessions are instantaneous and cease once the request is over? (Just making sure I read this right)

Reply:
1. I thought that this should be avoided, but now I see that it should not. Thanks!
2. That is the case, yeah!